components/claim-mgt/org.wso2.carbon.identity.claim.metadata.mgt/src/main/java/org/wso2/carbon/identity/claim/metadata/mgt/cache/LocalClaimCache.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.claim.metadata.mgt.cache;
+
+import org.wso2.carbon.identity.claim.metadata.mgt.model.LocalClaim;
+
+import java.util.ArrayList;
+
+/**
+ * Cache implementation for LocalClaims.
+ */
+public class LocalClaimCache extends BaseCache<Integer, ArrayList<LocalClaim>> {
+
+    private static final LocalClaimCache instance = new LocalClaimCache();
+    private static final String CACHE_NAME = "LocalClaimCache";
+
+    private LocalClaimCache() {
+        super(CACHE_NAME);
+    }
+
+    public static LocalClaimCache getInstance() {
+        return instance;
+    }
+
```

Comment:
Remove unwanted line

Reply:
fixed in https://github.com/wso2/carbon-identity-framework/pull/2474